src/sos-packaging.props

```diff
@@ -5,98 +5,98 @@
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm.$(Configuration)\libsosplugin.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm.$(Configuration)\libsos.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm</PackagePath>
     </None>
 
     <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm64.$(Configuration)\libsosplugin.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm64.$(Configuration)\libsos.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\Linux.arm64.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-arm64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/linux-arm64</PackagePath>
     </None>
 
     <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/osx-x64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\libsosplugin.dylib" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/osx-x64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\libsos.dylib" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/osx-x64</PackagePath>
     </None>
     <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/osx-x64</PackagePath>
```

Comment:
Defining this Variable now seems like a contract to whoever imports this file. Can we create a target that hangs before pack or at least a comment where it's used so that it's not a pitfall during cleanups?

Reply:
Added comment documenting this input variable.